commands/admin.js

```diff
@@ -141,20 +152,20 @@ exports.demote = function(message, args) {
     }
     var id_rx = /^<@([0-9]+)>$/g;
     var id = id_rx.exec(args[0]);
-    console.log(id);
-    id = id ? id[1] : args[0]
-    console.log(id);
+    id = id ? id[1] : args[0];
     var user = global.util.getUser({id: id});
+    logger.info(`Locating user ${id}`);
     if(user === undefined || global.userList[user.id] === undefined || global.userList[user.id]>=3 || global.userList[user.id]<=0){
-        message.channel.send(`Cannot demote that user!`)
+        message.channel.send(`Cannot demote that user!`);
         return;
     }
+    logger.info(`Demoting ${user.username}`);
     global.database.demoteUser(user).then(
         function(results){
-            global.util.updateUserPermissions(message.channel, `Successfully demoted ${args[0]}`);
+            global.util.updateUserPermissions(message.channel, `Successfully demoted ${user}`);
         }).catch(
         function(reason){
-            console.log(reason);
+            logger.error(reason);
             message.channel.send(`Update Failed, see system logs`);

```

Comment:
If this and line 169 are specific error messages to on command can we change the term update to the specific command. Promote Failed/Demote Failed